README.rst

```diff
@@ -2,10 +2,10 @@
 TOML
 ****
 
-.. image:: https://badge.fury.io/py/toml.svg
-    :target: https://badge.fury.io/py/toml
+.. image:: https://img.shields.io/pypi/v/toml
+    :target: https://pypi.org/project/toml/
 
-.. image:: https://travis-ci.org/uiri/toml.svg?branch=master
```

Comment:
Hi!

Thanks for the pull request! I think the change from "fury.io" to "shields.io" and pypi is worthwhile.
I would prefer to keep using the badge directly from travis-ci, not least for the ability to specify the default branch used.

Reply:
@uiri I respect your preference. Changed this back as you requested